computing-resources/eduroam.md

```diff
@@ -0,0 +1,43 @@
+# <span>📶</span> Eduroam
+
+## Overview
+
+As a student or employee of Polytechnique Montreal, you are granted access to the ['eduroam'](https://eduroam.org/what-is-eduroam/) network. 
+
+> ### What is eduroam?
+> 
+> eduroam (education roaming) is the secure, world-wide roaming access service developed for the international research and education community.
+> 
+> eduroam allows students, researchers and staff from participating institutions to obtain Internet connectivity across campus and when visiting other participating institutions by simply opening their laptop.
+
+When on campus, you may see a WiFi network called 'eduroam'. However, to use this network, some extra setup is required. (You can't just log in with your PolyMTL email address and password.)
+
+## Setup
+
+PolyMTL [recommends](https://www.polymtl.ca/si/reseaux/reseau-sans-fil#WIFI_Personnel_Etudiant) using the ["Eduroam CAT tool"](https://cat.eduroam.org/) to set up the network connection on each of your devices. This tool is available for Windows, macOS, and Linux, as well as Android and iOS.
```

Comment:
Personally, I do not trust giving this sketchy script `sudo` access to my system, but it _is_ the official poly recommendation, so :+1: from me for the sake of consistency.